docker-compose.yml

```diff
@@ -144,6 +146,17 @@ services:
       POSTGRES_HOST_AUTH_METHOD: "trust"
     volumes:
       - "sentry-postgres:/var/lib/postgresql/data"
+  pgcat:
+    image: "$PGCAT_IMAGE"
+    ports:
+      - "5432:5432"
```

Comment:
We need to remove this. Nothing should be exposed other than `$SENTRY_BIND:80` on `nginx` container.

Reply:
It's okay. I usually debug self-hosted Sentry using GitHub Actions